lib/suse/connect/client.rb

```diff
@@ -83,6 +90,7 @@ def flatten_tree(tree)
       #
       # @returns: [Array] login, password tuple. Those credentials are given by SCC/Registration Proxy
       def announce_system(distro_target = nil, instance_data_file = nil)
+        log.info "Announcing system to SCC/RMT/SMT...\n".bold
```

Comment:
Was 'SCC/RMT/SMT' blessed? Would 'registration server' be clearer to an outsider who is not familar with all of our registration proxies?

Reply:
Not blessed, just invented :angel: